util/cosmos/iscn/sign.js

```diff
@@ -107,7 +107,15 @@ function preformatISCNPayload(payload) {
   }
 
   const contentFingerprints = [];
-  if (fingerprint) contentFingerprints.push(`ipfs://${fingerprint}`);
+  if (fingerprint) {
+    if (Array.isArray(fingerprint)) {
+      for (let i = 0; i < fingerprint.length; i += 1) {
+        contentFingerprints.push(fingerprint[i]);
```

Comment:
How about `.concat()`?

Reply:
Also what if a old plugin version directly send ipfs hash without `ipfs://` ? This will break iscn formatting